### PR TITLE
adding pulp_rpm and 2to3 migration bindings

### PR DESCRIPTION
**Pull request by jlsherrill**
for katello

### CODE REVIEW

**Comment by jlsherrill** [comment]
This should be the last set for a long while!

**Comment by ehelms** [comment]
You'll want to add them to comps as well. I can handle adding the new package names to the koji tag unless you want to.

**Comment by jlsherrill** [comment]
@ehelms thanks! i knew i missed a reference, went ahead and add-pkg'd to koji

**Comment by ekohl** [comment]
@jlsherrill https://github.com/theforeman/foreman-packaging/blob/rpm/develop/rel-eng/compare-with-tags.rb uses comps to tell you the koji commands. It's for people like me who are lazy.

**Comment by jlsherrill** [comment]
[test rpm]

**Comment by jlsherrill** [comment]
Thanks @ekohl, updated!

File: packages/katello/rubygem-pulp_rpm_client/rubygem-pulp_rpm_client.spec [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby >= 1.9
+Requires: %{?scl_prefix_ruby}ruby(rubygems) > 1.3.1
+Requires: %{?scl_prefix}rubygem(faraday) >= 0.14.0
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1
+Requires: %{?scl_prefix_ruby}rubygem(json) < 3
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1.0
```
Comment (ehelms) [inline packages/katello/rubygem-pulp_rpm_client/rubygem-pulp_rpm_client.spec]: Triple json requires?

File: packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby >= 1.9
+Requires: %{?scl_prefix_ruby}ruby(rubygems) > 1.3.1
+Requires: %{?scl_prefix}rubygem(faraday) >= 0.14.0
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1
+Requires: %{?scl_prefix_ruby}rubygem(json) < 3
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1.0
```
Comment (ehelms) [inline packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec]: Similar question on the json requires

File: packages/katello/rubygem-pulp_rpm_client/rubygem-pulp_rpm_client.spec [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+%global gem_name pulp_rpm_client
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 3.0.0b7.dev01570381057
+Release: 1%{?dist}
+Summary: Pulp 3 RPM plugin API Ruby Gem
+Group: Development/Languages
+License: GPLv2
+URL: https://github.com/pulp/pulp_rpm/
```
Comment (ekohl) [inline packages/katello/rubygem-pulp_rpm_client/rubygem-pulp_rpm_client.spec]: It is confusing to not find sources at this URL.

File: packages/katello/rubygem-pulp_rpm_client/rubygem-pulp_rpm_client.spec (hunk 1 above)
Comment (jlsherrill) [reply packages/katello/rubygem-pulp_rpm_client/rubygem-pulp_rpm_client.spec]: was auto-generated, but i will remove one

File: packages/katello/rubygem-pulp_rpm_client/rubygem-pulp_rpm_client.spec (hunk 3 above)
Comment (jlsherrill) [reply packages/katello/rubygem-pulp_rpm_client/rubygem-pulp_rpm_client.spec]: there is no source location for the bindings, as they are generated from the api specification defined in the pulp_rpm repository

File: packages/katello/rubygem-pulp_rpm_client/rubygem-pulp_rpm_client.spec (hunk 1 above)
Comment (jlsherrill) [reply packages/katello/rubygem-pulp_rpm_client/rubygem-pulp_rpm_client.spec]: updated

**Pull request merged**